src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechRecognizer.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Set;
 import java.util.TreeSet;
 
+import org.apache.tools.ant.taskdefs.LoadProperties;

```

Comment:
Seems like a strange library to be importing,, is there a better import that is more generic?

Reply:
dunno how that got pulled in .. but all this code is "Outdated" - no longer an import